float64 num_a
float64 num_b
---
float64 result
